### PR TITLE
add relinquishment test WINFF.FT.S.RLQ.9

### PR DESCRIPTION
**Pull request by qingwu123**

### CODE REVIEW

File: src/harness/testcases/relinquishment_testcase.py [hunk 1]
```diff
@@ -229,6 +229,106 @@ def test_WINFF_FT_S_RLQ_7(self):
...
+    The response should be FAIL.
+    """
+
+    # Register the device
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2['fccId'] = "test_fcc_id_2"
+    device_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_4['fccId'] = "test_fcc_id_4"
```
Comment (kate-harrison) [inline src/harness/testcases/relinquishment_testcase.py]: Why not just load device_b.json and avoid having to create new FCC IDs?

File: src/harness/testcases/relinquishment_testcase.py [hunk 2]
```diff
@@ -229,6 +229,106 @@ def test_WINFF_FT_S_RLQ_7(self):
...
+    self.assertFalse('grantId' in response[1])
+    self.assertEqual(response[1]['response']['responseCode'], 103)
+    self.assertFalse('cbsdId' in response[2])
+    self.assertFalse('grantId' in response[2])
+    self.assertIn(response[2]['response']['responseCode'], [103, 105])
+    self.assertEqual(response[3]['cbsdId'], cbsd_id_2)
+    self.assertEqual(response[3]['grantId'], grant_id_4)
+    self.assertEqual(response[3]['response']['responseCode'], 103)
+    self.assertEqual(response[4]['cbsdId'], cbsd_id_2)
+    self.assertFalse('grantId' in response[4])
```
Comment (kate-harrison) [inline src/harness/testcases/relinquishment_testcase.py]: I think this is supposed to be equal to grant_id_3?

File: src/harness/testcases/relinquishment_testcase.py (hunk 1 above)
Comment (qingwu123) [inline src/harness/testcases/relinquishment_testcase.py]: Sure, I can do that.

File: src/harness/testcases/relinquishment_testcase.py (hunk 2 above)
Comment (qingwu123) [inline src/harness/testcases/relinquishment_testcase.py]: grand_id_3 is relinquished earlier so it's no longer a valid grant. TS says invalid grant id should not be included in the response. I have requested Masoud/Kambiz to make correct in the test plan.

**Comment by qingwu123** [comment]
It definitely felt like tongue twister equivalent of a test case.

**Pull request merged**